src/main/java/GoblintResult.java

```diff
@@ -65,7 +71,7 @@
                 piece.loc.column - 1,
                 findColumnEnd(piece.loc.line, piece.loc.column), 
                 sourcefileURL);
-                GoblintAnalysisResult result = new GoblintAnalysisResult(pos, "Group: " + multipiece.group_text, piece.text, severity);
+                GoblintAnalysisResult result = new GoblintAnalysisResult(pos, "Group: " + findTags() + multipiece.group_text, piece.text, severity);
```

Comment:
Goblint's terminal output has group tags before group text. Maybe this should also be the case here?